app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt

```diff
@@ -110,7 +110,10 @@ class ChipClient @Inject constructor(@ApplicationContext context: Context) {
               continuation.resumeWithException(error)
             }
           })
+      
+      // Temporary workaround to remove interface indexes from ipAddress
```

Comment:
In the comment, we should add a link to https://github.com/project-chip/connectedhomeip/pull/19394/files, and explain that the fix to establishPaseConnection was overlooked.

Reply:
Added